communications/newsletters/newsletter_02_Oct2019.md

```diff
@@ -0,0 +1,87 @@
+
+### Collaboration Cafes online call series
+
+Ever since their launch in September 2019, total 4 online Collaboration Cafes have already taken place. :tada:
+
+Thanks to all our attendees for coming along and joining us to learn more about these calls, tips on community management, GitHub tricks and the project itself. You can watch the recording from these calls on our [YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA).
+
+Read all about this online call series on [this page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md). We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you.
+
+Thanks to [Paula Andrea Martinez](https://github.com/orchid00), National Characterisation Training Coordinator at the Australian National Imaging Facility, we've already added an "Australia-friendly" cafe on 30 October. :sparkling_heart:
+
+If you live in a country that is not compatible with our [current schedule](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#attending-an-online-collaboration-cafe) and if you would like to join a call in the future, please add your comment in [this GitHub issue](https://github.com/alan-turing-institute/the-turing-way/issues/711).
+
+There's an issue open to collected your ideas ([#711](https://github.com/alan-turing-institute/the-turing-way/issues/711)).
+Thank you in advance!
+
+## Relevant resources
+
+### Open Life Science Mentoring Program
+
+The Turing Way is delighted to collaborate with the [Open Life Science Program](https://openlifesci.org) organised by Bérénice Batut, Yo Yehudi and Malvika Sharan as one of the [Mozilla Open Leader X projects](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/). They are developing this mentoring program for early-stage researchers who want to become ambassadors for Open Life Science in their communities. A call for application for mentees and mentors will be announced soon. Stay up to date through their webpage (https://openlifesci.org/).
+
+## Tips & Tricks for new contributors
+
+Are you new to this project and want to participate in the community? You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.
+
+See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of ['good first pull' issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.
```

Comment:
```suggestion
See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of [good first issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.
```
Just removed pull if that's ok because I think there are lots of ways of contributing without making a PR...